### PR TITLE
Insert only impala load template

### PR DESCRIPTION
**Pull request by BoBaeva**
A new template is created that only inserts new rows into the dw tables. It is created from an already existing one that overwrites rows for a period of time specified in the view. This template is needed because we maintain large tables that the platform cannot handle when they are regularly populated and our data jobs fail.

### CODE REVIEW

File: projects/vdk-plugins/vdk-impala/src/vdk/plugin/impala/templates/load/fact/insert/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+```python
+def run(job_input):
+    # . . .
+    template_args = {
+        'source_schema': 'default',
+        'source_view': 'vw_fact_vmc_utilization_cpu_mem_every5min_daily',
+        'target_schema': 'history',
+        'target_table': 'fact_vmc_utilization_cpu_mem_every5min_daily',
+        'last_arrival_ts': 'updated_at',
```
Comment (murphp15) [inline projects/vdk-plugins/vdk-impala/src/vdk/plugin/impala/templates/load/fact/insert/README.md:36]: Should this property be here? 
its not in the template args section.

Comment (BoBaeva) [reply projects/vdk-plugins/vdk-impala/src/vdk/plugin/impala/templates/load/fact/insert/README.md]: Thanks, @murphp15! I removed the argument from template_args.

File: projects/vdk-plugins/vdk-impala/tests/functional/jobs/insert_template_partition_job/__init__.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Copyright 2021 VMware, Inc.
```
Comment (tozka) [inline projects/vdk-plugins/vdk-impala/tests/functional/jobs/insert_template_partition_job/__init__.py:1]: Please remove the empty `__init__.py` files. I think those are created accidentally by VSCode

**Review by tozka — APPROVED** [review]
Looks good to me. Only remove the empty __init__.py files.

**Pull request closed without merging**